src/test/java/gov/cms/madie/terminology/service/VsacServiceTest.java

```diff
@@ -592,4 +592,34 @@ void testGetCodeStatusIfCodeNotFoundInSvs() {
     CodeStatus status = vsacService.getCodeStatus(code, TEST_API_KEY);
     assertThat(status, is(equalTo(CodeStatus.NA)));
   }
+
+  @Test
+  void testUserUmlsLogout() {
+    when(umlsUserRepository.findByHarpId(anyString())).thenReturn(Optional.of(umlsUser));
+    UmlsUser user = vsacService.verifyUmlsAccess(TEST_API_KEY);
```

Comment:
I think this is unnecessary 
`vsacService.verifyUmlsAccess(TEST_API_KEY);`

Reply:
@RohitKandimalla removed